assets/js/blocks/single-product-details/index.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+
+registerBlockType( metadata, {
```

Comment:
We should make the block available only on the Single Product Template. I created a [util function](https://github.com/woocommerce/woocommerce-blocks/blob/e526eff91a4bd6e955e62793f5b18c259305e5ce/assets/js/atomic/utils/register-block-single-product-template.ts) to do this.

Reply:
I think that function is not merged into trunk yet, right?